cookbook/category/add-properties-to-category.rst

```diff
@@ -5,38 +5,48 @@ The Akeneo PIM allows the classification of products inside a customizable categ
 
 .. note::
 
-    To implement this task you have to be comfortable with Symfony bundle overriding and Symfony services.
+    To implement this task you have to be comfortable with Symfony bundle overriding and Symfony services. This cookbook can be used to override other entities as attribute.
```

Comment:
means we can override other entities than a category, for instance, an attribute